scripts/deploy_heroku.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+# Get the latest image from Docker Hub (built by your CI pipeline) 
+docker pull sm4o/todo.app:latest
```

Comment:
This should still be available on the VM from the previous step, but no harm in having it!